### PR TITLE
ISPN-11549 Upgrade to 11

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-11549

### CODE REVIEW

**Comment by pruivo** [comment]
800 lines just to upgrade to ISPN 11? damn...

**Comment by karesti** [comment]
I added the tutorials that were moved

File: spring-integration/spring-boot/embedded/pom.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    </parent>
+
+    <artifactId>infinispan-spring-boot-sample-embedded</artifactId>
+    <name>Infinispan Simple Tutorials: Spring Boot Embedded</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-spring-boot-starter-embedded</artifactId>
+            <version>2.3.0.Final</version>
```
Comment (pruivo) [inline spring-integration/spring-boot/embedded/pom.xml:19]: use `version.infinispan.spring.boot`

File: spring-integration/spring/spring-session/pom.xml [hunk 2]
```diff
@@ -33,7 +28,7 @@
         <dependency>
             <groupId>org.infinispan</groupId>
             <artifactId>infinispan-spring-boot-starter-embedded</artifactId>
-            <version>2.2.3.Final</version>
+            <version>2.3.0.Final</version>
```
Comment (pruivo) [inline spring-integration/spring/spring-session/pom.xml:31]: use `version.infinispan.spring.boot`

File: spring-integration/spring-boot/embedded/pom.xml (hunk 1 above)
Comment (karesti) [reply spring-integration/spring-boot/embedded/pom.xml:19]: done

File: spring-integration/spring/spring-session/pom.xml (hunk 2 above)
Comment (karesti) [reply spring-integration/spring/spring-session/pom.xml:31]: done

**Comment by karesti** [comment]
@pruivo should be good now

**Comment by pruivo** [comment]
integrated! thanks @karesti !

**Pull request merged**